requirements-tests.txt

```diff
@@ -2,15 +2,9 @@
 -r requirements.txt
 
 pytest
-pytest-timeout
-pytest-xdist
 pylint
 flake8
-flake8-per-file-ignores
-psutil
-mock==2.0.0.
-hypothesis
 coverage
-codecov
-timeout-decorator
```

Comment:
were that removed packages excessive?

Reply:
Thanks for catching this!  I messed something up with the requirements.  Well, it's a good opportunity to test if they are needed actually :-)  I re-added those which are actually used.